tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AdoNet/DatabaseMonitoringPropagator.cs

```diff
@@ -0,0 +1,40 @@
+// <copyright file="DatabaseMonitoringPropagator.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.AdoNet
+{
+    internal static class DatabaseMonitoringPropagator
+    {
+        private const string SqlCommentSpanService = "dddbs";
+        private const string SqlCommentRootService = "ddps";
+        private const string SqlCommentVersion = "ddpv";
+        private const string SqlCommentEnv = "dde";
+
+        private const string SqlCommentTraceParent = "traceparent";
+
+        internal static string PropagateSpanData(string propagationStyle, string configuredServiceName, Span span)
+        {
+            var propgationComment =
+                $"{SqlCommentRootService}='{configuredServiceName}'," +
+                $"{SqlCommentSpanService}='{span.ServiceName}'," +
+                $"{SqlCommentVersion}='{span.GetTag(Tags.Version)}'," +
+                $"{SqlCommentEnv}='{span.GetTag(Tags.Env)}'";
```

Comment:
Nitpicking - Those string interpolations will happen in the hotpath. So the less allocations you do the better. I don't recall if this is optimal TBH. Naturally I would have moved the const part of the interpolations into a static string. In some cases (like context propagation), we use a StringBuilder instead.
So this may be good enough, I'm more raising the comment to let you know that if you think of things like that, in case of doubt you can copy your code in [sharplab](https://sharplab.io/) to see what happens behind the scene or even run micro benchmarks (we have a micro benchmarks [here](https://github.com/DataDog/dd-trace-dotnet/tree/a72b74fc8e67d8d8a6430628fe8643b3a693d2bc/tracer/test/benchmarks/Benchmarks.Trace/microbenchmarks) where you can run some easily)

Reply:
I did some researching regarding the interpolation and ended using the above based on the details I found, I didn't know about that website and will looking into it!